library/foreman_provisioner.py

```diff
@@ -192,7 +194,10 @@ def ipmi(self, host_id, command):
         commands: 'status', 'on', 'off', 'cycle', 'reset', 'soft' # TBD
         """
         command = "ipmitool -I lanplus -H {0} -U ADMIN -P ADMIN chassis power {1}".format(host_id, command)
-        response = os.system(command)
+        return_code = subprocess.call(command.split(" "))
```

Comment:
If you already the command string you can set the 'shell' flag to 'True' instead of splitting the command.
```python
return_code = subprocess.call(command, shell=True)
```